### PR TITLE
Gee, it sure is boring around here. I wonder what the Syndicate is up to?

### PR DESCRIPTION
**Pull request by Iamgoofball**
My boy, this peace is what all true maintainers strive for.

:cl: Iamgoofball
add: adds a flying carpet to ride
/:cl:

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
>My boy, this peace is what all true maintainers strive for

u fokin wot you got the quote wrong goddamn its "this place" fuck

**Comment by Iamgoofball** [comment]
no m8 he's talking about things being boring, the king replies that this PEACE is what all true heroes strive for, because peace is good

fucking let's hit the subtitles

**Comment by PKPenguin321** [comment]
this is really really funny, not just for the reference

**Comment by Buggy123** [comment]
> someone says goof would code a magic carpet if someone sprited it
 > hours later goof codes magic carpet

**Comment by Iamgoofball** [comment]
someone asked me to make a magic carpet and I told adminss to tell OOC I'd do it if they made me sprites, then kor came up with the idea of "what if its a shuttle that blocks air movement"

**Review by praisenarsie — CHANGES_REQUESTED** [review]
mapmerge

**Comment by 81Denton** [comment]
mah boi

**Comment by 81Denton** [comment]
Can't wait for this thing to launch and fling 90% of the crew into space

**Comment by Ordonis** [comment]
>Tfw a goofcode storm comes and and the PRs are actually entertaining and the titles vaguely relate to the code involved

**Comment by 81Denton** [comment]
@Ordonis HMMMMM I wonder if there are some sort of elections going on right now

File: _maps/shuttles/emergency_carpet.dmm [hunk 1]
```diff
@@ -0,0 +1,19 @@
+"a" = (/turf/template_noop,/area/template_noop)
```
Comment (ShizCalev) [inline _maps/shuttles/emergency_carpet.dmm]: TGM.

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#mapping-standards

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/shuttles/emergency_carpet.dmm [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+"b" = (/turf/open/floor/carpet/magic,/area/shuttle/escape)
+"c" = (/obj/structure/chair/comfy/brown{dir = 4},/turf/open/floor/carpet/magic,/area/shuttle/escape)
+"d" = (/obj/machinery/computer/emergency_shuttle{dir = 8},/turf/open/floor/carpet/magic,/area/shuttle/escape)
+"e" = (/obj/docking_port/mobile/emergency{dheight = 0; dir = 2; dwidth = 9; name = "Magic Carpet"; width = 19},/turf/open/floor/carpet/magic,/area/shuttle/escape)
```
Comment (ShizCalev) [inline _maps/shuttles/emergency_carpet.dmm]: dheight = 0 is a dirty var.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/datums/shuttles.dm [hunk 3]
```diff
@@ -199,6 +199,12 @@
 	description = "A small, but feature complete shuttle. It boasts a card table to keep crew members occupied on the long flight home."
 	credit_cost = 1000
 
+/datum/map_template/shuttle/emergency/carpet
+	suffix = "carpet"
+	name = "Magic Carpet"
+	description = "Shadala, we're off!"
```
Comment (McBawbaggings) [inline code/datums/shuttles.dm]: Isn't it "Squadala" though?

File: _maps/shuttles/emergency_carpet.dmm (hunk 1 above)
Comment (Iamgoofball) [reply _maps/shuttles/emergency_carpet.dmm]: Having python issues but will do

**Comment by Potato-Masher** [comment]
Squadala, we're off!

File: code/datums/shuttles.dm (hunk 3 above)
Comment (Armhulen) [reply code/datums/shuttles.dm]: it is, he won't fix it >:(

Comment (81Denton) [reply code/datums/shuttles.dm]: booo

**Comment by KorPhaeron** [comment]
@iamgoofball

This needs to not throw people on liftoff

**Comment by Iamgoofball** [comment]
i feel like that's part of the experience

wait didnt we already disable that

**Comment by JJRcop** [comment]
I won't let #35963 be forgotten!

**Comment by KorPhaeron** [comment]
@iamgoofball

The experience is gonna be people jostling during the flight if it just straight up kills everyone on launch admins will disable it

**Comment by ExcessiveUseOfCobblestone** [comment]
only one can survive, kor...

**Comment by Iamgoofball** [comment]
If only one can survive, I'm putting my money on Mr. Rodgers.

**Comment by MrDoomBringer** [comment]
@Iamgoofball his sweater would probably be bloodstained tho

**Comment by optimumtact** [comment]
dumb idea like the maker of the pr

**Comment by Iamgoofball** [comment]
my python is fucked and I cant mapmerge anyways

**Pull request closed without merging**